### ISSUE TITLE
CommonWA spec

### ISSUE THREAD

**Issue by oskbor** [Geal/serverless-wasm#1]
There seems to be a spec in the works here:
https://github.com/CommonWA/cwa-spec

**Comment by Geal** [comment]
yes! As you can probably see, I've started to contribute to that spec and will update my API :)
